src/Lecture1.hs

```diff
@@ -66,8 +69,8 @@ sumOfSquares x y = error "TODO!"
 🕯 HINT: use the @mod@ function
 
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integral a => a -> a
+lastDigit = (`mod` 10) . abs
```

Comment:
Nice usage of composition! Though, in this shape it might be a bit harder to read 😅

Reply:
I've updated the solution